cognite/neat/rules/to_rdf_path.py

```diff
@@ -142,11 +142,10 @@ def from_string(cls, raw: str, **kwargs) -> Self:
 
 
 class Traversal(BaseModel):
-    pass
```

Comment:
I hope this works, just hope we do not relive January nightmares

Reply:
I checked it fairly carefully. so think it should be fine

Reply:
I did an extra check of all the usage, and the fact that `traversal.class_` was already assumed almost all the places it was used. In addition, this is adding variable to a class (all subclasses are unchanged since they already have it). So I think it is very safe to do this change.